src/utils/traits/to_str.rs

```diff
@@ -0,0 +1,13 @@
+pub trait ToStr {
+  fn to_str(self) -> &'static str;
+}
+
+impl ToStr for bool {
+  fn to_str(self) -> &'static str {
```

Comment:
Small trait to remove the need to allocate a string on the heap when stringifying bools.